### PR TITLE
[Modular] Snails & Blushing Fix (again!)

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
Snails now have recolorable eyes again. Spacemen now have a blushing overlay again.

## How This Contributes To The Skyrat Roleplay Experience
Snussy.

## Changelog
:cl:
fix: Snails now have recolorable eyes again.
fix: Blushing now works again.
/:cl:

### CODE REVIEW

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Don't change human_face.dmi directly, change it in a modular file and link the snail eyes to use that file instead.

**Comment by Gandalf2k15** [comment]
Have you checked to see if this is a problem upstream?

**Comment by Nerev4r** [comment]
> Have you checked to see if this is a problem upstream?

It isn't.

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -384,6 +384,11 @@ GLOBAL_LIST_EMPTY(customizable_races)
 						eye_emissive.pixel_y += species_human.dna.species.offset_features[OFFSET_FACE][2]
 						standing += eye_emissive
 
+		// blush
+		if (HAS_TRAIT(species_human, TRAIT_BLUSHING)) // Caused by either the *blush emote or the "drunk" mood event
+			var/mutable_appearance/blush_overlay = mutable_appearance('icons/mob/human_face.dmi', "blush", -BODY_ADJ_LAYER) //should appear behind the eyes
+			blush_overlay.color = COLOR_BLUSH_PINK
+			standing += blush_overlay
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm:391]: How does this decide whether or not you're having those visuals on your character? Is it handled by the emote itself not giving you the TRAIT_BLUSHING if you haven't got the pref on? I just want to be sure this isn't forced on everyone if they have the pref for it off.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm (hunk 1 above)
Comment (Nerev4r) [reply modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm:391]: Doesn't have a pref, it's a default /tg/ behavior https://github.com/tgstation/tgstation/pull/66321

**Pull request merged**